### ISSUE TITLE
Removal of old reports does not work properly if DEST_DIR is set

### ISSUE THREAD

**Issue by bamanczak** [PavanMudigonda/allure-html-reporter-aws-s3-website#2]
Removal of old reports does not take into account the `DEST_DIR`. I belive the error lies in the following lines:
https://github.com/PavanMudigonda/allure-html-reporter-aws-s3-website/blob/main/entrypoint.sh#L140-L143

Similarly, the `index.html` that should show the list of previous runs, shows the list of folders from root of the bucket, instead of list of folders from `DEST_DIR`

**Comment by kotmatpockuh** [comment]
Did you fix this issue @bamanczak ?

**Comment by bamanczak** [comment]
No, I just ended up using a different solution (https://github.com/andrcuns/allure-publish-action)
